### PR TITLE
Correct `ownProps` note and examples

### PR DESCRIPTION
**Pull request by erykpiast**
I hope it's true this time.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -72,25 +72,39 @@ Instead, it *returns* a new, connected component class, for you to use.
   * [`pure = true`] *(Boolean)*: If true, implements `shouldComponentUpdate` and shallowly compares the result of `mergeProps`, preventing unnecessary updates, assuming that the component is a “pure” component and does not rely on any input or state other than its props and the selected Redux store’s state. *Defaults to `true`.*
   * [`withRef = false`] *(Boolean)*: If true, stores a ref to the wrapped component instance and makes it available via `getWrappedInstance()` method. *Defaults to `false`.*
 
-> Note: `ownProps` is passed to `mapStateToProps` and `mapDispatchToProps` only if formal definition of the function contains two mandatory parameters (function length has to be greater or equal 2). For example, functions defined like below won't receive `ownProps` as the second argument.
+> Note: `ownProps` **is not passed** to `mapStateToProps` and `mapDispatchToProps` if formal definition of the function contains one mandatory parameter (function has length 1). For example, function defined like below won't receive `ownProps` as the second argument.
 ```javascript
-function mapStateToProps() {
-  console.log(arguments[0]); // state
+function mapDispatchToProps(state) {
```
Comment (gaearon) [inline docs/api.md]: Should be `mapStateToProps`

Comment (erykpiast) [inline docs/api.md]: I think it's worth to show that this is about `mapDispatchToProps` too, that's why I did it. You would have it consistent I guess?

Comment (gaearon) [inline docs/api.md]: The confusing part is the argument is still named "state". I would probably just use mapStateToProps in both examples.

**Comment by gaearon** [comment]
Thanks!

**Pull request merged**